### PR TITLE
Migrate from intltool to gettext

### PR DESCRIPTION
**Pull request by yetist**

### CODE REVIEW

File: libmate-panel-applet/org.mate.panel.TestApplet.mate-panel-applet.desktop.in [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Applet Factory]
+Id=TestAppletFactory
+Location=test-dbus-applet
+Name=Test DBus Applet Factory
+Description=Factory for Test DBus Applet
+
+[TestApplet]
+Name=Test DBus Applet
+Description=A simple applet for testing the MATE panel
+Icon=mate-gegl
```
Comment (raveit65) [inline libmate-panel-applet/org.mate.panel.TestApplet.mate-panel-applet.desktop.in]: We need translator hints for icon name, or not?

File: mate-panel/mate-panel.desktop.in [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Type=Application
-_Name=Panel
-_Comment=Launch other applications and provide various utilities to manage windows, show the time, etc.
+Name=Panel
+Comment=Launch other applications and provide various utilities to manage windows, show the time, etc.
 Icon=mate-panel
```
Comment (raveit65) [inline mate-panel/mate-panel.desktop.in]: Same here

Comment (raveit65) [reply mate-panel/mate-panel.desktop.in]: and there are `Categories=GTK;System;Core;` in this file.

**Comment by raveit65** [comment]
@yetist 
We have desktop files with with translated strings of icon-names. Translator hints are needed.
eg.
```
#: applets/clock/org.mate.panel.ClockApplet.mate-panel-applet.desktop.in.in:12
msgid "mate-panel-clock"
msgstr ""

#: applets/fish/org.mate.panel.FishApplet.mate-panel-applet.desktop.in.in:12
msgid "mate-panel-fish"
msgstr ""

#: applets/notification_area/org.mate.panel.NotificationAreaApplet.mate-panel-applet.desktop.in.in:12
msgid "mate-panel-notification-area"
msgstr ""

#: applets/wncklet/org.mate.panel.Wncklet.mate-panel-applet.desktop.in.in:12
msgid "mate-panel-window-menu"
msgstr ""

**Comment by raveit65** [comment]
I only squashed last commit in first one.

**Comment by raveit65** [comment]
@yetist 
We have a problem, some UI elements are not translated. I will post more tomorrow, now it is to late here.

**Comment by yetist** [comment]
> We have a problem, some UI elements are not translated. I will post more tomorrow, now it is to late here.

Which UI elements?

**Comment by yetist** [comment]
@raveit65 

I confirm that.
```
$ strace -o logs mate-panel --replace
$ grep "\.mo" logs |grep mate-panel              
openat(AT_FDCWD, "/usr//locale/zh_CN.UTF-8/LC_MESSAGES/mate-panel.mo", O_RDONLY) = -1 ENOENT (没有那个文件或目录)
openat(AT_FDCWD, "/usr//locale/zh_CN.utf8/LC_MESSAGES/mate-panel.mo", O_RDONLY) = -1 ENOENT (没有那个文件或目录)
openat(AT_FDCWD, "/usr//locale/zh_CN/LC_MESSAGES/mate-panel.mo", O_RDONLY) = -1 ENOENT (没有那个文件或目录)
openat(AT_FDCWD, "/usr//locale/zh.UTF-8/LC_MESSAGES/mate-panel.mo", O_RDONLY) = -1 ENOENT (没有那个文件或目录)
openat(AT_FDCWD, "/usr//locale/zh.utf8/LC_MESSAGES/mate-panel.mo", O_RDONLY) = -1 ENOENT (没有那个文件或目录)
openat(AT_FDCWD, "/usr//locale/zh/LC_MESSAGES/mate-panel.mo", O_RDONLY) = -1 ENOENT (没有那个文件或目录)

$ ls -l /usr/share/locale/*/LC_MESSAGES/mate-panel.mo|wc -l
133
```
Each mo file is placed in the right location, but when running mate-panel, it does not look for the mo file in the correct location.

**Pull request merged**